sycl/include/sycl/info/aspects_deprecated.def

```diff
@@ -0,0 +1,4 @@
+__SYCL_ASPECT_DEPRECATED(int64_base_atomics, 7, "use atomic64 instead")
+__SYCL_ASPECT_DEPRECATED(int64_extended_atomics, 8, "use atomic64 instead")
+// Special macro for aspects that don't have own token
+__SYCL_ASPECT_DEPRECATED_ALIAS(usm_system_allocator, usm_system_allocations, "use usm_system_allocations instead")
```

Comment:
nit: I'd personally avoid having two files, but I don't have strong argumentation.

Reply:
@steffenlarsen , I'm not sure if you're "thumbing up" my "I'd avoid" or "don't have strong argumentation".

I'm approving the PR to unblock @KornevNikita but if you'd prefer to have a single file as well we can ask him to address that :) Both pre-/post-commit would work for me.

Reply:
Both, really. Having a single file would be a little nicer, but I don't think it's worth sweating over. 😄